h2o-core/src/main/java/hex/grid/GridSearch.java

```diff
@@ -65,23 +65,28 @@
  * Model[] models = grid.getModels()
  * }</pre>
  *
- * @see hex.grid.HyperSpaceWalker
+ * @see HyperSpaceWalker
  * @see #startGridSearch(Key, HyperSpaceWalker)
  */
 public final class GridSearch<MP extends Model.Parameters> extends Keyed<GridSearch> {
   public final Key<Grid> _result;
   public final Job<Grid> _job;
+  public final int _parallelismLevel;
+
+  public static final int SEQUENTIAL_MODE = 1;
+  public static final int ADAPTIVE_PARALLEL_MODE = 0;
```

Comment:
Can't you use an enum instead here? Also, how is it different from `GridSearchStrategy` enum?
It's a bit annoying when having methods with many parameters to figure out what this `int` is... without mentioning the better error detection when using an enum.